qgreenland/config/datasets/seal_tag_water_measurements.py

```diff
@@ -0,0 +1,54 @@
+from qgreenland.models.config.asset import HttpAsset
+from qgreenland.models.config.dataset import Dataset
+
+dataset = Dataset(
+    id="seal_tag_measurements",
+    assets=[
+        HttpAsset(
```

Comment:
This dataset has just a single file (asset) we're interested in pulling (the `ct71_0DV.csv`). Note  there are other files associated with the dataset on the ADC landing page, but we don't need those in QGreenland (e.g., a browse .png image).